src/main/java/hudson/plugins/brakeman/scanners/AbstractBrakemanScanner.java

```diff
@@ -0,0 +1,16 @@
+package hudson.plugins.brakeman.scanners;
+
+import hudson.plugins.analysis.core.ParserResult;
+
+/**
+ * A Java class that representats a JSON Scanner of the Brakeman Output file
```

Comment:
This is an interface and not a class.  There is a big difference between those two in the java world.  There are suggestions below to create a common scanner super class.  If you end up taking that advice just convert this sucker to an abstract class with a few common concert methods and mark `scan(String content, ParserResult project)` and the class as `abstract`.

if you keep this as an interface:
- update the javadoc to refer to it as an interface and not a class,
- change the name from AbstractBrakemanScanner to BrakemanScanner ... interfaces typically shouldn't start with a prefix of `Abstract`.